requirements/requirements.txt

```diff
@@ -0,0 +1,7 @@
+neo>=0.8.0,<0.9.0
+elephant>=0.6.4
```

Comment:
please check if it works with elephant-0.6.4 and elephant-0.7.0

Reply:
My dev environment for this release was elephant==0.6.4. I have tested now by changing it to elephant==0.7.0 and plot_isi, plot_corrcoef and plot_ue all work like they should.